cmd/probe/main.go

```diff
@@ -19,22 +20,25 @@ var (
 )
 
 func main() {
-	flag.StringVar(&url, "url", "", "URL or host:port to connect to")
-	flag.DurationVar(&timeout, "timeout", 5*time.Second, "Timeout value (e.g. 5s, 1m)")
-	flag.IntVar(&expectedStatus, "expected-status", 200, "Expected HTTP status code (0 for connection failure)")
-	flag.StringVar(&testType, "type", "http", "Type of test (http or tcp)")
-	flag.BoolVar(&expectFail, "expect-fail", false, "Whether the test is expected to fail (TCP tests only)")
+	flag.StringVar(&url, pf.ArgUrl, "", "URL or host:port to connect to")
```

Comment:
See comment in `pkg/probeflags/probeflags.go`:
```suggestion
	flag.StringVar(&url, pf.ArgURL, "", "URL or host:port to connect to")
```